client/payment-methods/index.js

```diff
@@ -278,6 +277,31 @@ const PaymentMethods = () => {
 					</CardHeader>
 				) }
 
+				{ isUpeEnabled && upeType === 'legacy' && (
+					<CardHeader className="payment-methods__header">
+						<InlineNotice
+							icon
+							status="warning"
+							isDismissible={ false }
+						>
+							<div>
+								{ __(
+									'The new WooPayments checkout experience will become the default on October 2, 2023.',
+									'woocommerce-payments'
+								) }
+								<br />
+								{ /* eslint-disable-next-line max-len */ }
+								<ExternalLink href="https://woocommerce.com/document/woopayments/payment-methods/additional-payment-methods/">
+									{ __(
+										'Learn more',
+										'woocommerce-payments'
+									) }
+								</ExternalLink>
+							</div>
```

Comment:
(minor) The line wrapping with the "learn more" seems inconsistent with other places in the app - what do you think about putting the "learn more" inline?
![Screenshot 2023-09-14 at 1 07 51 PM](https://github.com/Automattic/woocommerce-payments/assets/273592/f0da7fcb-c141-4b53-a984-9f7ad265c330)


```suggestion
							{ interpolateComponents( {
								mixedString: __(
									'The new WooPayments checkout experience will become the default on October 2, 2023.' +
										' {{learnMoreLink}}Learn more{{/learnMoreLink}}',
									'woocommerce-payments'
								),
								components: {
									learnMoreLink: (
										// eslint-disable-next-line max-len
										<ExternalLink href="https://woocommerce.com/document/woopayments/payment-methods/additional-payment-methods/" />
									),
								},
							} ) }
```

![Screenshot 2023-09-14 at 1 08 01 PM](https://github.com/Automattic/woocommerce-payments/assets/273592/4c86f659-f980-49c6-a678-e2a5aef102dd)

Reply:
Makes total sense! https://github.com/Automattic/woocommerce-payments/pull/7210/commits/d5892742a15f1c8b5e7809ac98bb5302aa65645e